packages/nc-gui/components/cell/Email.vue

```diff
@@ -86,9 +86,9 @@ watch(
 
   <span v-else-if="vModel === null && showNull" class="nc-null">NULL</span>
 
-  <a v-else-if="validEmail" class="text-sm underline hover:opacity-75" :href="`mailto:${vModel}`" target="_blank">
+  <nuxt-link v-else-if="validEmail" class="text-sm underline hover:opacity-75 inline-block" :href="`mailto:${vModel}`" target="_blank">
```

Comment:
why `inline-block` is required here?

Reply:
The clickable area in email used to take up the entire cell. Here inline-block is used to make sure the link only takes up the space occupied by the email text.

Reply:
got it.